hathor/consensus/block_consensus.py

```diff
@@ -216,7 +216,9 @@ def update_voided_info(self, block: Block) -> None:
                         if common_block not in heads:
                             self.context.mark_as_reorg(common_block)
                 else:
-                    storage.update_best_block_tips_cache([not_none(blk.hash) for blk in heads])
+                    best_block_tips = [not_none(blk.hash) for blk in heads]
+                    best_block_tips.append(not_none(block.hash))
+                    storage.update_best_block_tips_cache(best_block_tips)
```

Comment:
Should we just set to `None` and let the next call cache it?

Reply:
Either way is fine for me. I wish we can get rid of this cache in the future. Will we even need it if we don't have best block ties, because even for sync-v1 the block height index would have a ready `O(1)` (or at least however fast `best_blocks[-1]` is) response?